public/data/teyddie22.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Teyddie",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/teyddie22.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/teyddie22",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/teyddie22",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "December 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing"
```

Comment:
```suggestion
      "description": "Started freelancing!"
```

Reply:
@krishguptadev exclamation marks should be their choice too 😉